src/cameraAccessory.ts

```diff
@@ -97,7 +97,15 @@ export class CameraAccessory {
       })
       .onSet((status) => {
         this.log.debug(`Setting alarm to ${status ? "on" : "off"}`);
-        this.camera.setAlertConfig(Boolean(status));
+        this.camera.setAlertConfig(Boolean(status))
```

Comment:
Hey, thanks for the PR; do you think you could use the `async/await` pattern instead and `try/catch` the entire block? thanks